### PR TITLE
[readme] update for rtfx

### PR DESCRIPTION
**Pull request by sanchit-gandhi**
Updates the README to include details on:
1. The joint computation of WER and RTFx
2. The machine specifications for the benchmark
3. A template script `run_eval.py` that can be copied and easily updated for new benchmark submissions

cc @Vaibhavs10 @nithinraok

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,38 +9,220 @@ Each library has its own set of requirements. We recommend using a clean conda e
...
 3) Install the common requirements for all library by running `pip install -r requirements/requirements.txt`.
-4) If you wish to run NeMo, note that the benchmark currently needs CUDA 12.6 (`nvidia-smi` should output "CUDA Version: 12.6" or higher), to fix a problem in previous drivers for RNN-T inference with cooperative kernels inside of conditional nodes (see here: https://github.com/NVIDIA/NeMo/pull/9869)
-5) Install the requirements for each library you wish to evalaute by running `pip install -r requirements/requirements_<library_name>.txt`.
-6) Connect your Hugging Face account by running `huggingface-cli login`.
+4) Install the requirements for each library you wish to evaluate by running `pip install -r requirements/requirements_<library_name>.txt`.
+5) Connect your Hugging Face account by running `huggingface-cli login`.
+
+**Note:** If you wish to run NeMo, the benchmark currently needs CUDA 12.6 to fix a problem in previous drivers for RNN-T inference with cooperative kernels inside conditional nodes (see here: https://github.com/NVIDIA/NeMo/pull/9869). Running `nvidia-smi` should output "CUDA Version: 12.6" or higher.
```
Comment (Vaibhavs10) [inline README.md:15]: Just for my understanding, we also ran our Transformers benchmarks on the CUDA 12.6 right?

Comment (sanchit-gandhi) [reply README.md:15]: Yep! Everything run on the same machine, same CUDA, same pytorch, same python, etc.

File: README.md [hunk 2]
```diff
@@ -9,38 +9,220 @@ Each library has its own set of requirements. We recommend using a clean conda e
...
+Each library has a script `run_eval.py` that acts as the entry point for evaluating a model. The script is run by the corresponding bash script for each model that is being evaluated. The script then outputs a JSONL file containing the predictions of the model on each dataset, and summarizes the Word Error Rate (WER) and Inverse Real-Time Factor (RTFx) of the model on each dataset after completion.
+
+To reproduce existing results:
 
 1) Change directory into the library you wish to evaluate. For example, `cd transformers`.
 2) Run the bash script for the model you wish to evaluate. For example, `bash run_wav2vec2.sh`.
-3) **Note**: All evaluations are done on single GPU. If you wish to run two scripts in parallel, please use `CUDA_VISIBLE_DEVICES=<0,1,...N-1>` prior to running the bash script, where `N` is the number of GPUs on your machine.
+
+**Note**: All evaluations were run using an NVIDIA A100-SXM4-80GB GPU, with NVIDIA driver 560.28.03, CUDA 12.6, and PyTorch 2.4.0. You should ensure you use the same configuration when submitting results. If you are unable to create an equivalent machine, please request one of the maintainers to run your scripts for evaluation!
```
Comment (sanchit-gandhi) [inline README.md:26]: See this line for details @Vaibhavs10

Comment (Vaibhavs10) [reply README.md:26]: Ah my bad! I'm blind! Thanks! <3

**Comment by sanchit-gandhi** [comment]
Corresponding Hub PR: https://huggingface.co/spaces/hf-audio/open_asr_leaderboard/discussions/17

**Pull request merged**